docs/src/SUMMARY.md

```diff
@@ -72,7 +72,7 @@
         * [parse_json](appendix/forge-library/fs/parse_json.md)
     * [env](appendix/forge-library/env.md)
         * [var](appendix/forge-library/env/var.md)
-    * [signature](appendix/forge-library/signature)
+    * [signature](appendix/forge-library/signature.md)
```

Comment:
👀

Reply:
Fixed in #955 too, it prevented docs from building xD

Reply:
Docs didn't build because of this 👀 from what I see they are not checked anywhere in CI

Reply:
#961